functional-test/src/test/groovy/org/rundeck/util/common/jobs/JobUtils.groovy

```diff
@@ -116,26 +118,38 @@ class JobUtils {
         return jobs[0]
     }
 
+
     static Execution waitForExecutionToBe(
             String state,
             String executionId,
             ObjectMapper mapper,
             RdClient client,
-            int iterationGap,
-            int timeout
+            WaitingTime iterationGap,
+            WaitingTime timeout
+    ) {
+        return waitForExecutionToBe(state, executionId, mapper, client, iterationGap.duration, timeout.duration)
+    }
+
+    static Execution waitForExecutionToBe(
+            String state,
+            String executionId,
```

Comment:
```suggestion
            int executionId,
```
Poor man's URL injection protection :-)

Reply:
I'm good with this change, but can we do it in a different PR? Honestly i don't want to go again over the 116 callers to check if they expect an int or a String.